ep006/README.md

```diff
@@ -0,0 +1,40 @@
+# [python type(x).__name__ vs x.__class__.__name__ (intermediate)](https://youtu.be/6rAIttnm3Fs)
+
+Very short one, I explain why I prefer type(x).__name__ vs x.__class__.__name__.  thanks to @alikus for the question!
```

Comment:
these will need backticks around them or they'll end up italicized

Reply:
Yep, I will extract the descriptions manually from now on. They need additional processing.